docs/technical_reference/unit_models/crystallizer_0D.rst

```diff
@@ -0,0 +1,121 @@
+
+Crystallization (0D)
+=======================
+
+Introduction
+-------------
+Crystallization from solution is a separation technique where a solid phase is separated from a mother
+liquor (Kramer et al. [1]).
+
+The model implemented here is for a continuous evaporative crystallization. The main features of the implemented model are as follows:
+
+1) Evaporative crystallization is modeled as shown in Figure 1.
+2) Model dimensionality is limited to 0D.
```

Comment:
Does "Model dimensionality is limited to 0D" have a specific, technical meaning here? Or are you basically just saying "this is a simple model for this unit"? If the latter, then I'd recommend rewording or removing this point, to avoid confusing the reader about what the meaning of "0D model" is.

Reply:
Thought about it a little more - does "0D" just mean that the model does not include a spatial component, in any dimensions?

Reply:
Yes, that's what it means.